mars/oscar/backends/ray/communication.py

```diff
@@ -284,3 +286,10 @@ async def connect(dest_address: str,
     @implements(Client.close)
     async def close(self):
         await super().close()
+
+
+def _gen_client_id():
+    import socket
+    import os
+    import time
+    return f"{socket.gethostname()}:{os.getpid()}:{time.time_ns()}"
```

Comment:
Hope the `time_ns()` provides enough precision. I prefer the uuid.

Reply:
Actually `{socket.gethostname()}:{os.getpid()}` will make this process unique. Using uuid is better